tools/codegen-gradle-plugin/README.md

```diff
@@ -0,0 +1,46 @@
+### VRAP Gradle plugin
+
+In order to make the integration of the code generator in a gradle build easy we created a gradle plugin with id
+`io.vrap.rmf.codegen-plugin` (not yet published to gradle central), in order to use it you have to specify the `uri`
+parameter with is the raml file location, plus targets
+
+
+A target is a generation target it contains
+
+   * serviceName: your service name
```

Comment:
What is a service name? May be we should call it `targetName`?

Reply:
service name means actually the module name for example `import-storage-service`....